pangeo_forge_recipes/patterns.py

```diff
@@ -63,12 +63,13 @@ class MergeDim(CombineDim):
     operation: ClassVar[CombineOp] = CombineOp.MERGE
 
 
-def augment_index_with_start_stop(
+def augment_index_with_byte_range(
```

Comment:
I believe this is indexing logical dimensions in the dataset, not bytes.

Reply:
It looks to me as though this function uses the logical position of the provided index to calculate start/stop in terms of bytes, no? Like, the sum of item lengths calculated prior to it is determined based on logical position but the resulting integer is supposed to be the byte-range start is how i read it